smart/map_processing.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+import skimage as ski
+from matplotlib import colors
+from skimage.morphology import disk
+
+import astropy.units as u
+
+from sunpy.map import Map, all_coordinates_from_map, coordinate_is_on_solar_disk
+
+
+def map_processing(file):
+    """
+    Creating a map from a fits file and processing said map.
+
+    Parameters
+    ----------
+    file : fits
+        fits file to turn into map.
+
+    Returns
+    -------
+    im_map : sunpy.map.Map
+        processed map created from fits file.
+
+    """
+    im_map = Map(file)
+    if im_map.meta["CROTA2"] != 0:
+        im_map = im_map.rotate(order=3)
```

Comment:
Still not totally clear why this is need if reprojecting they should be the same?